packages/flame_tiled/README.md

```diff
@@ -1,6 +1,10 @@
 # flame_tiled
 
-> :warning: Under the current sprite batch implementation, Flips are not working prperly. Any help is appreciated looking into this issue.
+> :warning: Under the current sprite batch implementation, Flips are not working properly. 
+> Any help is appreciated looking into this issue.
```

Comment:
Shouldn't there be link to the issue?

Reply:
I'm not sure there is one, I haven't been involved in this (I only made a line break here). 
We definitely should have an issue for it though, but someone that knows more about this should probably create it instead of me.